Automattic/wp-calypso
# Jetpack onboarding: set up initial structure for the Summary section of the flow

JPO Summary step

This PR tweaks the very basics:
- add document head
- add follow-up redirect Button, its temporary redirect href and set centered positioning

**To test:**
- checkout this branch
- go to `http://calypso.localhost:3000/jetpack/onboarding/summary`
and verify that the `Visit your site` Button is centered and redirects to `/jetpack/onboarding/site-title`

![screen shot 2017-12-12 at 00 21 30](https://user-images.githubusercontent.com/13561163/33860757-3586397a-ded2-11e7-8170-9e1406cef9a0.png)
